### PR TITLE
Add test to the visual monitor.

### PR DESCRIPTION
**Pull request by mldk-tech**
#1

### CODE REVIEW

**Comment by mldk-tech** [comment]
@amitaibu - notice the design breaks on 940px

ss

**Comment by amitaibu** [comment]
:+1:

File: visual-monitor/test/tests.js [hunk 1]
```diff
@@ -61,12 +50,23 @@ describe('Visual monitor testing', function() {
...
+        exclude:
+          [
+            // Teachers and students.
+            '.hom_col .ttm_image',
+            '.hom_col .ttm_text',
+          ],
+        hide:
+          [
+            // Sidebar.
+            '.hom_col .hom_skills',
```
Comment (amitaibu) [inline visual-monitor/test/tests.js]: lets keep the sidebar please

File: visual-monitor/test/tests.js [hunk 2]
```diff
@@ -61,12 +50,23 @@ describe('Visual monitor testing', function() {
...
+          ],
+        screenWidth: selectedCaps == 'chrome' ? [1200] : undefined,
```
Comment (amitaibu) [inline visual-monitor/test/tests.js]: lets use 960 by default instead of the 1200 (just looks better in the screenshots I do later from within shoov)

**Comment by mldk-tech** [comment]
@amitaibu - Ok, done.

**Pull request merged**